lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -0,0 +1,275 @@
+              The Complete Development Environment for Elixir and Phoenix
+            </p>
+          </div>
+
+          <div class="md:flex lg:w-[75%] flex-col relative">
+            <div class="md:flex">
+              <div class="flex justify-end items-center md:order-last md:mx-2 sm:mb-2">
+                <Heroicons.information_circle
+                  class="w-6 h-6 text-white cursor-pointer focus:text-gray-500"
+                  phx-click={JS.show(to: "#installation-instructions")}
+                />
+              </div>
+              <div class="border sm:px-3 bg-[#26168780] w-full flex items-center justify-center py-2 sm:space-x-3 rounded-sm overflow-x-auto whitespace-nowrap no-scrollbar">
+                <h1 id="tool-installation" class="text-center text-white font-martian sm:text-sm">
+                  <%= render_slot(@installation_command) %>
+                </h1>
+                <div id="copy" phx-hook="CopyHook" class="sm:py-3 md:my-0 cursor-pointer">
+                  <Icons.copied_icon />
+                  <Icons.copy_icon />
+                </div>
+              </div>
+            </div>
+            <div class="w-full bg-blue-600 flex justify-end">
+              <.os_instructions live_action={@live_action} operating_system={@operating_system} />
+            </div>
+            <div class="justify-center gap-4 md:flex">
+              <div class="sm:grid grid-cols-2 gap-4 sm:py-4">
+                <.command_select_button
+                  live_action={@live_action}
+                  operating_system={@operating_system}
+                />
+                <.source_code_button source_code_url={@source_code_url} />
+              </div>
+            </div>
+          </div>
+          <div class="lg:w-[1100px]">
+            <div>
+              <p class="text-xs px-2 leading-5 text-center text-white font-martian md:mb-5 md:text-sm md:leading-6 lg:text-base">
+                Phx.tools is a shell script for Linux and macOS that configures the development environment for you in a few easy steps. Once you finish running the script, you'll be able to start the database server, create a new Phoenix application, and launch the server.
+              </p>
+            </div>
+            <div
+              class="mx-auto"
+              data-asciicast={
+                if @live_action == :macOS,
+                  do: "bJMOlPe5F4mFLY0Rl6fiJSOp3",
+                  else: "XhDpRstBJ4df2gfiRfp0awDPO"
+              }
+              id={"asciinema-#{Utils.random_id()}"}
+              phx-hook="AsciinemaHook"
+            >
+            </div>
+
+            <p class="md:my-5 text-white md:px-12 text-xs md:text-sm lg:text-base font-martian">
+              Read about website updates here -
+              <a
+                href="https://optimum.ba/blog/phx-tools-complete-development-environment-for-elixir-and-phoenix"
```

Comment:
The newest blog post will reside at https://optimum.ba/blog/exciting-updates-to-phx-tools.